lib/bolt/analytics.rb

```diff
@@ -139,6 +141,15 @@ def report_bundled_content(mode, name)
         end
       end
 
+      def plan_counts(project_path)
+        pp_count, yaml_count = %w[pp yaml].map do |extension|
+          Find.find(File.join(project_path, 'plans')).grep(/.*\.#{extension}/).length
```

Comment:
This fails if there's no `plans` directory in the project. We should probably both accommodate that and wrap this in an error handler that returns `{}` so analytics don't cause a crash.

Reply:
Hmm how does it fail? I don't seem to be able to reproduce that...
```
lucy ~/githubs/bolt (GH-2302) $ ls | grep 'plan'
lucy ~/githubs/bolt (GH-2302) $ be bolt command run whoami -t localhost
Started on localhost...
Finished on localhost:
  STDOUT:
    lucy
Successful on 1 target: localhost
Ran on 1 target in 0.01 sec
lucy ~/githubs/bolt (GH-2302) $ ls | grep 'bolt-project.yaml'
bolt-project.yaml
```

Reply:
`bundle exec` disables analytics by default, is that why it's not failing for you?